README.md

```diff
@@ -1,8 +1,7 @@
 # Biblioteca de integração PagSeguro em Ruby
 
-[![Build Status](https://travis-ci.org/pagseguro/ruby.png?branch=master)](https://travis-ci.org/pagseguro/ruby)
-
-[![Code Climate](https://codeclimate.com/github/pagseguro/ruby.png)](https://codeclimate.com/github/pagseguro/ruby)
+[![Build Status](https://travis-ci.org/pagseguro/ruby.svg?branch=2.3.0-dev-update-transaction-by-code)](https://travis-ci.org/pagseguro/ruby)
```

Comment:
Well… can you change from:

* `https://travis-ci.org/pagseguro/ruby.svg?branch=2.3.0-dev-update-transaction-by-code` to;
* `https://travis-ci.org/pagseguro/ruby.svg?branch=master`

?